src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/SqlAction.cs

```diff
@@ -467,6 +467,56 @@ NpgsqlTransaction transaction
         return result;
     }
 
+    /// <summary>
+    /// Update the ReferentialId of a document by its DocumentUuid for cases
+    /// when identity updates are permitted. 
+    /// </summary>
+    public static async Task<int> UpdateAliasReferentialIdByDocumentUuid(
+        short referentialPartitionKey,
+        Guid referentialId,
+        short documentPartitionKey,
+        Guid documentUuid,
+        NpgsqlConnection connection,
+        NpgsqlTransaction transaction
+    )
+    {
+        var result = await GetPostgresExceptionRetryPipeline().ExecuteAsync(async _ =>
+        {
+            try
+            {
+                await using var command = new NpgsqlCommand(
+                    @"UPDATE dms.Alias AS a
```

Comment:
Does this work if a document references the document before the identity change? Doesn't the `Reference` table need updating? I don't see an update cascade setting on the FK in the 1000 SQL script, maybe that's all you need?